src/components/tooltip/Tooltip.tsx

```diff
@@ -60,8 +61,25 @@ export interface TooltipProps extends Omit<React.HTMLProps<HTMLDivElement>, 'as'
   type?: 'hint' | 'tooltip';
   /** if true, the tooltip should be displayed */
   visible?: boolean;
+  debounceTime?: number;
```

Comment:
The name imho is very "implementation linked", we talk more about a hoverTimeout here I think
Also a comment for storybook is welcome :)